index.js

```diff
@@ -189,8 +190,8 @@ const sleepingTime = 1800000;
         } catch (e) {
             console.log('Routine error at: ', new Date().toLocaleString(), e)
         }
-        await console.log(process.env.ACCOUNT,'waiting for the next battle in ', sleepingTime / 1000 / 60, ' minutes')
-        await console.log('If you need support for the bot join the telegram group https://t.me/splinterlandsbot and dont pay scammers')
+        await console.log(process.env.ACCOUNT,'waiting for the next battle in',sleepingTime / 1000 / 60,'minutes at', new Date(Date.now() +sleepingTime).toLocaleString() )
```

Comment:
can you please add the space before and after the minutes in the console.log?

Reply:
`        await console.log(process.env.ACCOUNT,'waiting for the next battle in', sleepingTime / 1000 / 60 , 'minutes at' , new Date(Date.now() +sleepingTime).toLocaleString() )
`
Like this?

Reply:
no i meant in the console message so:
sleepingTime / 1000 / 60, ' minutes at ', new Date(Date.now() +sleepingTime).toLocaleString() )